packages/rs-dpp/src/state_transition/state_transitions/identity/identity_create_transition/mod.rs

```diff
@@ -68,6 +68,23 @@ impl IdentityCreateTransition {
             }),
         }
     }
+
+    pub fn get_minimal_asset_lock_value(
```

Comment:
Why this method in `IdentityCreateTransition`? There is a module related to asset locks in src/identity/ you could put it there (it can be just a module function).